### ISSUE TITLE
GromacsWrapper causes conda actions failed

### ISSUE THREAD

**Issue by AnguseZhang** [deepmodeling/dpgen#614]
See https://github.com/deepmodeling/dpgen/runs/4484676453?check_suite_focus=true

Solution:
1. Fix conda action. I did publish successfully with `GromacsWrapper` in other packages. So don't know what's wrong with `linux-64` platform.
2. Not include `GromacsWrapper` in offline package. But this will cause users cannot use DP-GEN with Gromacs.
3. Modify codes and remove `GromacsWrapper` dependecies.

**Comment by AnguseZhang** [comment]
@njzjz What do you think of？

**Comment by njzjz** [comment]
Did you upload to our channel?

**Comment by AnguseZhang** [comment]
> 

No. `GromacsWrapper` should have already been in `conda-forge` channel.

**Comment by njzjz** [comment]
I think `conda-forge` does not have `GromacsWrapper` - https://anaconda.org/conda-forge/GromacsWrapper cannot be visited.

**Comment by AnguseZhang** [comment]
> I think `conda-forge` does not have `GromacsWrapper` - https://anaconda.org/conda-forge/GromacsWrapper cannot be visited.

You're right. I just created a PR https://github.com/deepmodeling/dpgen/pull/615
